### ISSUE TITLE
ItemHandler needs to support shiny stats

### ISSUE THREAD

**Issue by magicus** [Wynntils/Wynntils#1934]
They can appear like this in the tooltip:
```
§c✖§7 Class Req: Mage/Dark Wizard§r
§c✖§7 Combat Lv. Min: 97§r
§c✖§7 Strength Min: 105§r
§f⬡ §7Mobs Killed: §f0§r
§a+25 §7Strength§r
§a+8% §7Thorns§r
```

We need to look for `§f⬡ §7.*: §f.*§r` in the item matcher; ignoring it when comparing two items to see if they are the same, but if it has changed, we need to send an event saying it has changed. 

This will need some design changes, since we have not had items which change their tooltip during use like this.

**Comment by geekazodium** [comment]
#1941 I wrote a tiny bit of code that reads the shiny stats.
(only did a few tests, regex is here https://regexr.com/7i5h5 )

**Comment by magicus** [comment]
This is fixed in #2000.
